### ISSUE TITLE
[Bug] Files don't open

### ISSUE THREAD

**Issue by hjek** [martinmimigames/little-file-explorer#70]
**Describe the bug**
When opening a file with another app through LFE, the other app is opened but not with the chosen file.

**To Reproduce**
1. Click on a file, say, a text file, `hello.md`
2. Click on **Open**
3. Select a text editor, say, Markor
4. Markor then opens, but with a new file, not `hello.md`

This happens with other files and apps too, for example opening a JPEG in Gallery after tapping it in LFE.

**Expected Behaviour**
I expected the selected file to be opened in the chosen app (like what happens in Amaze or Material Files).

**Versioning  information**
- App Version: 4.13
- OS: Android 11

**Comment by Cwpute** [comment]
Which Gallery app are you using exactly @hjek ?
i am having a similar problem i reported in issue #66 
Have you tested this behavior with other apps ?

**Comment by hjek** [comment]
> Which Gallery app are you using exactly @hjek ?

[This one.](https://f-droid.org/en/packages/com.dot.gallery)

**Comment by hjek** [comment]
> Have you tested this behavior with other apps ?

Yes. A few work with LFE, most fail silently, but at least Krita and Squircle CE show error messages about the file not existing where the file path is not the actual path of the file but a `content://` path.
